src/renderer/initializers/welcome-menu-registry.ts

```diff
@@ -19,22 +19,17 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-import { catalogURL, preferencesURL } from "../../common/routes";
+import { catalogURL } from "../../common/routes";
 import { WelcomeMenuRegistry } from "../../extensions/registries";
 import { navigate } from "../navigation";
 
 export function initWelcomeMenuRegistry() {
   WelcomeMenuRegistry.getInstance()
     .add([
       {
-        title: "Browse Your Catalog",
+        title: "Browse Clusters",
         icon: "view_list",
-        click: () => navigate(catalogURL())
-      },
-      {
-        title: "Configure Preferences",
-        icon: "settings",
-        click: () => navigate(preferencesURL())
+        click: () => navigate(catalogURL({ params: { group: "entity.k8slens.dev", kind: "KubernetesCluster" } } ))
       }
```

Comment:
Do we not want this here anymore?

Reply:
nope, we. decided to only have one call to action that is the most important thing for the user to do right now

Reply:
Okay